Cargo.toml

```diff
@@ -7,7 +7,7 @@ description = "Library with types for handling PCI devices"
 categories = ["hardware-support", "no-std"]
 readme = "README.md"
 license = "MIT/Apache-2.0"
-edition = "2018"
+edition = "2021"
```

Comment:
The 2021 edition has TryInto in the prelude. If preferred I can manually import it instead.